index.html

```diff
@@ -17,6 +17,9 @@
 		<link rel="stylesheet" href="css/reveal.min.css">
 		<link rel="stylesheet" href="css/theme/alley.css" id="theme">
 
+		<!-- Fonts -->
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic|Open+Sans+Condensed:300,700">
```

Comment:
I _thought_ you could [drop the `https://` completely](https://www.paulirish.com/2010/the-protocol-relative-url/) to have a protocol-relative URL on stuff like google fonts, but [the internet](https://jeremywagner.me/blog/stop-using-the-protocol-relative-url) says otherwise